src/index.css

```diff
@@ -87,26 +87,51 @@ header {
   display: flex;
   flex-direction: column;
   gap: 4rem;
-  width: 80%;
+  width: 88%;
   padding-top: 2.5rem;
 }
 
 #books-list {
   display: flex;
   flex-direction: column;
   gap: 1rem;
+  list-style-type: none;
 }
 
 .book {
   display: flex;
   padding: 2.5rem 2rem;
   background-color: #fff;
   border: 1px solid #e8e8e8;
-  justify-content: space-between;
+  gap: 4rem;
 }
 
 .b-info {
   min-width: 400px;
+  font-family: "Roboto Slab", serif;
+}
+
+.b-category {
+  display: block;
+  font-family: "Montserrat", sans-serif;
+  font-size: 1.05rem;
+  font-weight: 600;
+  font-stretch: normal;
+  font-style: normal;
+  line-height: normal;
+  letter-spacing: normal;
+  color: hsla(0, 0%, 7%, 0.45);
+  margin-bottom: 0.5rem;
+}
+
+.b-author {
+  font-size: 1rem;
+  font-weight: 300;
+  font-stretch: normal;
+  font-style: normal;
+  line-height: normal;
+  letter-spacing: normal;
+  color: #4386bf;
 }
 
 #add-book {
```

Comment:
Please be sure to give a bit of margin to the form so that your app resembles as much as possible to the given design 🤓

![image](https://user-images.githubusercontent.com/15807343/185769343-8dc879f4-7010-4a4f-ac7d-b086fcac50a5.png)

Reply:
@mateo951 Thank you for pointing this out! Done :heavy_check_mark: